notebooks/try-chroma.py

```diff
@@ -0,0 +1,136 @@
+import marimo
+
+__generated_with = "0.13.11"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import marimo as mo
+    import numpy as np
+    import polars as pl
+    import chromadb
+    return chromadb, mo, np, pl
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(
+        r"""
+    # Try out Chromadb for finding similar sentences
+
+    Using the sample sentence corpora and embeddings generated in previous steps.
+    """
+    )
+    return
+
+
+@app.cell
+def _(chromadb):
+    chroma_client = chromadb.EphemeralClient()  # using EphemeralClient for in-memory, since this is a small test-set of content
+
+    collection = chroma_client.get_or_create_collection(name="marx_test")
+    return (collection,)
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(
+        r"""Load sample sentence corpora and embeddings, and load into the chroma collection."""
+    )
+    return
+
+
+@app.cell
+def _(np, pl):
+    dnz_sentences = np.load(
+        "data/sentence-embeddings/dnz-sample-sents.npy", allow_pickle=True
```

Comment:
```suggestion
        "data/sentence-embeddings/dnz-sample-sents.npy"
```
I don't think you need to set this...